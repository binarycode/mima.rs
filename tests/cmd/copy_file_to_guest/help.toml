args = 'help copy-file-to-guest'
status.code = 0
