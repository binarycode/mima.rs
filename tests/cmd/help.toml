args = 'help'
status.code = 0
fs.sandbox = true
