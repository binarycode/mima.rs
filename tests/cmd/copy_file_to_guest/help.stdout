Copy file to guest

Usage: mima copy-file-to-guest <GUEST_ID> <PATH> [FILE_NAME]

Arguments:
  <GUEST_ID>   Guest ID
  <PATH>       File path
  [FILE_NAME]  Destination file name

Options:
  -h, --help  Print help
